x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations.tsx

```diff
@@ -0,0 +1,164 @@
+  const [query, setQuery] = useState(EuiSearchBar.Query.MATCH_ALL);
+  const [error, setError] = useState<EuiSearchBarOnChangeArgs['error'] | null>(null);
+  const [isFlyoutVisible, setIsFlyoutVisible] = useState<boolean>(false);
+  const [selectedModuleId, setSelectedModuleId] = useState<string | undefined>();
+
+  const closeFlyout = () => setIsFlyoutVisible(false);
+
+  /**
+   * Loads recognizer module configuration.
+   */
+  const loadModules = useCallback(async () => {
+    setIsLoading(true);
+    try {
+      const modulesReponse = (await getDataRecognizerModule()) as Module[];
+      setModules(modulesReponse);
+      setIsLoading(false);
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
+      setIsLoading(false);
+    }
+  }, [getDataRecognizerModule]);
+
+  useEffect(() => {
+    loadModules();
+  }, [loadModules]);
+
+  const filters: SearchFilterConfig[] = useMemo(
+    () => [
+      {
+        type: 'field_value_selection',
+        field: 'tags',
+        name: 'Tags',
+        filterWith: 'includes',
+        multiSelect: 'or',
+        options: TAGS.map((tag) => {
+          return {
+            value: tag,
+            view: tag,
+          };
+        }),
+      },
+    ],
+    []
+  );
+
+  const schema = {
+    strict: true,
+    fields: {
+      tags: {
+        type: 'string',
+        validate: (value: string) => {
+          if (!TAGS.some((tag) => tag === value)) {
+            throw new Error(`unknown tag (possible values: ${TAGS.map((tag) => tag)})`);
+          }
+        },
+      },
+    },
+  };
+
+  const filteredModules = useMemo(() => {
+    const clauses = query?.ast?.clauses ?? [];
+    return clauses.length > 0 ? filterModules(modules, clauses) : modules;
+  }, [query, modules]);
+
+  const onChange = ({ query: onChangeQuery, error: onChangeError }: EuiSearchBarOnChangeArgs) => {
+    if (onChangeError) {
+      setError(error);
+    } else {
+      setError(null);
+      setQuery(onChangeQuery);
+    }
+  };
+
+  if (isLoading === true) return <LoadingIndicator />;
+
+  return error ? (
+    <>
+      <EuiCallOut iconType="faceSad" color="danger" title={`Invalid search: ${error?.message}`} />
+      <EuiSpacer size="l" />
+    </>
+  ) : (
+    <>
+      <EuiSearchBar
+        defaultQuery={query}
+        box={{
+          placeholder: 'Search supplied configurations',
+          incremental: true,
+          schema,
+        }}
+        filters={filters}
+        onChange={onChange}
+      />
+      <EuiSpacer size="l" />
+      <EuiFlexGrid gutterSize="l" columns={3}>
+        {filteredModules.map(({ description, id, logo, title }) => {
+          return (
+            <EuiFlexItem key={id}>
+              <EuiCard
+                layout="horizontal"
+                icon={<EuiIcon size="xxl" type={isLogoObject(logo) ? logo.icon : logo} />}
+                title={title}
+                // isDisabled={false}
```

Comment:
Is this commented out line still needed?

Reply:
Removed in 9838277f8bb24511c72c3af4da087cbdb52624e7